### ISSUE TITLE
Enabling the Toolbox-Module while disabling one of the boxes leads to a crash client-side

### ISSUE THREAD

**Issue by Husky110** [codetaylor/artisan-worktables#135]
As stated in discord:
"okay, some more tinkering revealed the culprit...
in fact it's the toolbox-config-file that causes the crash.
if you enable the module, but disable any of the toolboxes - the client goes down."

**Comment by codetaylor** [comment]
Crash log? Versions?

**Comment by Husky110** [comment]
Version: 1.12.2
Crashlog:
```
---- Minecraft Crash Report ----

WARNING: coremods are present:
  BNBGamingCore (BNBGamingCore-1.12.2-0.10.0.jar)
Contact their authors BEFORE contacting forge

// Ouch. That hurt :(

Time: 10/15/18 2:08 PM
Description: Initializing game

java.lang.NullPointerException: Block Block{null} has null registry name
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:864)
	at com.codetaylor.mc.athenaeum.util.ModelRegistrationHelper.registerBlockItemModel(ModelRegistrationHelper.java:81)
	at com.codetaylor.mc.artisanworktables.modules.toolbox.ModuleToolbox.lambda$onClientRegister$0(ModuleToolbox.java:71)
	at com.codetaylor.mc.artisanworktables.modules.toolbox.ModuleToolbox$$Lambda$147/1905377675.register(Unknown Source)
	at com.codetaylor.mc.athenaeum.registry.RegistryEventHandler.onClientRegisterModelsEvent(RegistryEventHandler.java:106)
	at com.codetaylor.mc.athenaeum.module.ModuleBase.onClientRegisterModelsEvent(ModuleBase.java:240)
	at com.codetaylor.mc.athenaeum.module.ModuleEventRouter.lambda$onClientRegisterModelsEvent$33(ModuleEventRouter.java:192)
	at com.codetaylor.mc.athenaeum.module.ModuleEventRouter$$Lambda$161/2119313812.accept(Unknown Source)
	at com.codetaylor.mc.athenaeum.module.ModuleEventRouter.fireEvent(ModuleEventRouter.java:202)
	at com.codetaylor.mc.athenaeum.module.ModuleEventRouter.onClientRegisterModelsEvent(ModuleEventRouter.java:192)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_9_ModuleEventRouter_onClientRegisterModelsEvent_ModelRegistryEvent.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
	at net.minecraftforge.fml.common.eventhandler.EventBus$1.invoke(EventBus.java:144)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182)
	at net.minecraftforge.fml.client.FMLClientHandler.fireSidedRegistryEvents(FMLClientHandler.java:1062)
	at net.minecraftforge.fml.common.FMLCommonHandler.fireSidedRegistryEvents(FMLCommonHandler.java:764)
	at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:629)
	at net.minecraftforge.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:252)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:466)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:377)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:864)
	at com.codetaylor.mc.athenaeum.util.ModelRegistrationHelper.registerBlockItemModel(ModelRegistrationHelper.java:81)
	at com.codetaylor.mc.artisanworktables.modules.toolbox.ModuleToolbox.lambda$onClientRegister$0(ModuleToolbox.java:71)
	at com.codetaylor.mc.artisanworktables.modules.toolbox.ModuleToolbox$$Lambda$147/1905377675.register(Unknown Source)
	at com.codetaylor.mc.athenaeum.registry.RegistryEventHandler.onClientRegisterModelsEvent(RegistryEventHandler.java:106)
	at com.codetaylor.mc.athenaeum.module.ModuleBase.onClientRegisterModelsEvent(ModuleBase.java:240)
	at com.codetaylor.mc.athenaeum.module.ModuleEventRouter.lambda$onClientRegisterModelsEvent$33(ModuleEventRouter.java:192)
	at com.codetaylor.mc.athenaeum.module.ModuleEventRouter$$Lambda$161/2119313812.accept(Unknown Source)
	at com.codetaylor.mc.athenaeum.module.ModuleEventRouter.fireEvent(ModuleEventRouter.java:202)
	at com.codetaylor.mc.athenaeum.module.ModuleEventRouter.onClientRegisterModelsEvent(ModuleEventRouter.java:192)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_9_ModuleEventRouter_onClientRegisterModelsEvent_ModelRegistryEvent.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
	at net.minecraftforge.fml.common.eventhandler.EventBus$1.invoke(EventBus.java:144)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182)
	at net.minecraftforge.fml.client.FMLClientHandler.fireSidedRegistryEvents(FMLClientHandler.java:1062)
	at net.minecraftforge.fml.common.FMLCommonHandler.fireSidedRegistryEvents(FMLCommonHandler.java:764)
	at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:629)
	at net.minecraftforge.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:252)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:466)

-- Initialization --
Details:
Stacktrace:
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:377)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.12.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 288867360 bytes (275 MB) / 972554240 bytes (927 MB) up to 7635730432 bytes (7282 MB)
	JVM Flags: 4 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx8192m -Xms256m -XX:PermSize=256m
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP 9.42 Powered by Forge 14.23.5.2770 15 mods loaded, 15 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored

	| State | ID                | Version       | Source                              | Signature                                |
	|:----- |:----------------- |:------------- |:----------------------------------- |:---------------------------------------- |
	| UCH   | minecraft         | 1.12.2        | minecraft.jar                       | None                                     |
	| UCH   | mcp               | 9.42          | minecraft.jar                       | None                                     |
	| UCH   | FML               | 8.0.99.99     | forge-1.12.2-14.23.5.2770.jar       | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| UCH   | forge             | 14.23.5.2770  | forge-1.12.2-14.23.5.2770.jar       | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| UCH   | bnbgamingcore     | 0.10.0        | minecraft.jar                       | None                                     |
	| UCH   | crafttweaker      | 4.1.11        | CraftTweaker2-1.12-4.1.11.jar       | None                                     |
	| UCH   | jei               | 4.12.1.217    | jei_1.12.2-4.12.1.217.jar           | None                                     |
	| UCH   | athenaeum         | 1.12.2-1.14.0 | athenaeum-1.12.2-1.14.0.jar         | None                                     |
	| UCH   | artisanworktables | 1.12.2-1.19.0 | artisanworktables-1.12.2-1.19.0.jar | None                                     |
	| UCH   | blockdropstweaker | 2.4           | BlockDropsTweaker-1.12.2-2.4.jar    | None                                     |
	| UCH   | bnbgaminglib      | 2.16.1        | BNBGamingLib-1.12.2-2.16.1.jar      | None                                     |
	| UCH   | ctgui             | 1.0.0         | CraftTweaker2-1.12-4.1.11.jar       | None                                     |
	| UCH   | crafttweakerjei   | 2.0.2         | CraftTweaker2-1.12-4.1.11.jar       | None                                     |
	| UCH   | lttweaker         | 1.1.14        | LootTableTweaker-1.12.2-1.1.14.jar  | d476d1b22b218a10d845928d1665d45fce301b27 |
	| UCH   | triumph           | 3.3.0         | Triumph-1.12.2-3.3.0.jar            | None                                     |

	Loaded coremods (and transformers): 
BNBGamingCore (BNBGamingCore-1.12.2-0.10.0.jar)
  com.bloodnbonesgaming.bnbgamingcore.core.BNBGamingCoreClassTransformer
	GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.6.0 NVIDIA 388.13' Renderer: 'GeForce GTX 960/PCIe/SSE2'
	Launched Version: forge-14.23.5.2770
	LWJGL: 2.9.4
	OpenGL: GeForce GTX 960/PCIe/SSE2 GL version 4.6.0 NVIDIA 388.13, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: 
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	CPU: 8x Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz
```
